backend/uclapi/dashboard/urls.py

```diff
@@ -9,5 +9,9 @@
     url(r'api/rename/$', rename_app),
     url(r'api/regen/$', regenerate_app_token),
     url(r'api/delete/$', delete_app),
+    url(r'api/setcallbackurl/$', set_callback_url),
+    url(r'api/setscope/roombookings/$', set_rb_scope),
+    url(r'api/setscope/timetable/$', set_timetable_scope),
+    url(r'api/setscope/uclu/$', set_uclu_scope),
```

Comment:
There are some issues with that (mainly that each scope is sort of hard coded even in the model) and it would require a massive if..elif...etc. block. We may need to discuss how to scope more intelligently.